text/87_interacting_with_stakeholders.md

```diff
@@ -39,19 +39,19 @@ When you're in school, the teachers tend to have more knowledge than you on what
+Similarly, when discussing the system or problems that you are having with your boss, put it into terms that they can understand.  Remember that they often have a broader view of the system than you do, and things that you consider very important may not be as important to them.  Conversely, they may not be aware of problems that you consider obvious.  Providing an open flow of information can be very helpful in both directions, allowing you as a tester to understand the system more holistically and channel your energy and time appropriately, while giving your supervisors knowledge of the nitty-gritty of the data, defects, and concerns you are finding in your testing.
 
 ## The Red-Yellow-Green Template
 
 One of the best ways I have found to communicate status to managers higher up on the totem pole is the red-yellow-green template.  This is not my invention, or even unique to the software industry, but has been proven useful throughout different fields because it allows you to provide a very high-level overview of status while still providing hooks to go deeper into detail if necessary.  Remember that managers will often not have the same granularity of knowledge of the system that you as a tester will have, and so putting it into simpler terms will allow the system to be intellectually manageable.
 
-If you're familiar with US traffic lights, you know that they consist of three lights: green meaning "go", yellow meaning "slow down", and red meaning "stop".  The standard OS X Windows manager uses a similar convention: green for "maximize", yellow for "minimize", and red for "close".  In each case, green means to there is nothing stopping you (maximum window size, drive quickly), yellow means that there are some impediments to progress (the light is about to change to red, the window is minimized), and red means that there are some more serious impediments (traffic is coming from the orthogonal direction, the window is closed).
+If you're familiar with US traffic lights, you know that they consist of three lights: green meaning "go", yellow meaning "slow down", and red meaning "stop".  The standard OS X Windows manager uses a similar convention: green for "maximize", yellow for "minimize", and red for "close".  In each case, green means there is nothing stopping you (maximum window size, drive quickly), yellow means that there are some impediments to progress (the light is about to change to red, the window is minimized), and red means that there are some more serious impediments (traffic is coming from the orthogonal direction, the window is closed).
```

Comment:
green means to there is nothing stopping you => green means there is nothing stopping you